packaging/ubuntu-16.04/tests/integrationtests

```diff
@@ -74,6 +74,9 @@ backends:
         - adt-local:
             username: ubuntu
             password: ubuntu
+prepare: |
```

Comment:
ah-ha! so there is a separate spread.yaml indeed and I wasn't going insane :smiley:

Reply:
Yeah, it's very confusing